who.go

```diff
@@ -1,9 +1,20 @@
 package who
 
-import "fmt"
+import "https://github.com/golang/go/tree/master/src/fmt" //edited by devrirobotik.com at 22.28 on 18.02.2019
 
 func WeAre(team string) {
-	fmt.Println(`
+	team := Person{
```

Comment:
`Person` bu sekilde kullanabilmemiz icin oncesinde [struct ](https://gobyexample.com/structs) olarak tanimlamiz gerekmez mi?

Reply:
`team`'in aslinda `person`'lardan olusan bir array olmasi daha mantikli geliyor bana, ne dersiniz?

Reply:
Evet. Copy-paste edecek çoğunluğu görmek istedim sadece.